lib/hammer_cli_foreman/bookmark.rb

```diff
@@ -15,7 +15,12 @@ class ListCommand < HammerCLIForeman::ListCommand
         field :owner_type, _('Owner Type')
       end
 
-      build_options
+      extend_with(HammerCLIForeman::CommandExtensions::WithoutDefaults.new)
+
+      build_options do |o|
```

Comment:
With current approach this block should be added for other CRUD commands for each resource that doesn't support taxonomies. Also, those options should vanish as soon as we fix the API docs.